content/motivation.md

```diff
@@ -101,31 +106,22 @@ mylib-1.3.3_wk_13.8.08.tgz      somecode-20.7.09.tgz
 
 ```{discussion} Why Git?
 We will use [Git](https://git-scm.com) to record snapshots of our work:
-- Easy to set up - use even by yourself with no server needed.
-- Very popular: chances are high you will need to contribute to somebody else's code which is tracked with Git.
-- Distributed: good backup, no single point of failure, you can track and clean-up changes offline, simplifies collaboration model for open-source projects.
-- Important platforms such as [GitHub](https://github.com), [GitLab](https://gitlab.com), and [Bitbucket](https://bitbucket.org)
+- **Easy to set up**: no server needed.
+- **Very popular**: chances are high you will need to contribute to somebody else's code which is tracked with Git.
+- **Distributed**: good backup, no single point of failure, you can track and
+  clean-up changes offline, simplifies collaboration model for open-source
+  projects.
+- Important **platforms** such as [GitHub](https://github.com), [GitLab](https://gitlab.com), and [Bitbucket](https://bitbucket.org)
   build on top of Git.
-- Many platforms build on top of [GitHub](https://github.com).
-- Sharing software and data is getting popular and required in research context
-  and [GitHub](https://github.com) is a popular platform for sharing software.
-- However, *Git is a four-handle, dual boiler espresso machine, not instant coffee.* (paraphrased) [source](https://stevebennett.me/2012/02/24/10-things-i-hate-about-git/).
-  Git isn't the most user friendly and has its design quirks but deep design
-  is great and is definitely the most popular and what you are most likely to
-  need to know. So we teach it.
-
-
-Why not [Subversion](https://subversion.apache.org)?
-- Subversion is centralized (one server, many clients) and requires setting up and maintaining a server.
-- You cannot easily clean-up your recorded snapshots (commits) before you share them.
-- Not easy to get contributions from external contributors.
-
-
-Why not [Mercurial](https://www.mercurial-scm.org)?
-- Mercurial: many Git concepts still apply. For that matter, most important
-  lesson is **how and why to use version control**, which applies to any system
-  with some changes.
-- Even if you use Mercurial chances are high you need to contribute to a code tracked by Git.
+
+However, any version control is better than no version control and it is OK to prefer a different tool than Git.
+
+Other tools:
+- [Subversion](https://subversion.apache.org)
+- [Mercurial](https://www.mercurial-scm.org)
+
+Interesting newcomer:
+- [Pijul](https://pijul.org/)
 ```
 
```

Comment:
What about acknowledging and discussing some difficulties?

```suggestion
## Difficulties of version control

Despite the benefits, let's be honest, there are some difficulties:

- One more thing to learn (It's probably worth it, will save you more time in the long run.  Basic career skill.)
- Difficult if some people don't want to use it (in the worst case, you can feed them versions)
- Advanced things can be difficult,a bit too many gotchas (basics are often enough, ask others for help when needed)
```

What other common difficulties are there we can acknowledge at the start??